test/proposal/lvm_helper_test.rb

```diff
@@ -307,7 +307,7 @@
 
         expect(lvs).to contain_exactly(
           an_object_with_fields(lv_name: "one", size: (9.GiB - 4.MiB).to_i),
-          an_object_with_fields(lv_name: "two", size: 9.GiB.to_i),
+          an_object_with_fields(lv_name: "two", size: (9.GiB - 4.MiB).to_i),
```

Comment:
so previously size was bug? now both "one" and "two" looks same.  So previously there was a bug? To be honest I am bit confused by this and I do not feel confident about change.